### ISSUE TITLE
NVDA Access site down? 

### ISSUE THREAD

**Issue by mysunnytime** [nvaccess/nvda#13263]
Hi there, I tried to download and use NVDA, but the whole site is not accessible with 502 Bad Gateway.

![image](https://user-images.githubusercontent.com/5469750/150395436-ff675502-c959-4f6c-9549-b4749fdef39b.png)

### Steps to reproduce:

access https://www.nvaccess.org/ from a browser

### Actual behavior:
502 Bad Gateway

### Expected behavior:
Show the site content.

**Comment by feerrenrut** [comment]
Thanks, this has already been resolved.
